test/ZNSRootRegistrar.test.ts

```diff
@@ -154,6 +161,123 @@ describe("ZNSRootRegistrar", () => {
     );
   });
 
+  it("Throws if env variable is invalid", async () => {
```

Comment:
these should be moved to DeployCampaign.test.ts
this is a test for RootRegistrar contract